test/test_mirror_prox.jl

```diff
@@ -239,7 +239,7 @@ end
   end
   @testset "restart_scheme=adaptive_localized" begin
     parameters = generate_mirror_prox_params(
-      iteration_limit = 750,
+      iteration_limit = 800,
```

Comment:
Changed from 2000 to 800, since that was sufficient for passing, and made a stricter test.